{{cookiecutter.repo_name}}/environment.yaml

```diff
@@ -3,10 +3,14 @@ channels:
     - conda-forge
 dependencies:
     - black=23.1.0
-    - nbqa=1.6.1
+    - python=3.10.9
+    - pre-commit=3.0.4
     {%- if cookiecutter.install_jupyter == 'yes' %}
     - jupyter=1.0.0
+    - nbqa=1.6.1
```

Comment:
I think it's a good idea to move "nbqa" into that conditional. However, if we do this, we also need to add a condition to the pre-commit config to avoid that the nbqa-tasks are executed when the dependency is not installed.

Reply:
Fixed with rebase onto new main

Reply:
Fixed in b5218e9

Reply:
Looks like this closes #26